packages/reassure-measure/src/testingLibrary.ts

```diff
@@ -0,0 +1,140 @@
+import { config, Render, Cleanup } from './config';
+
+type TestingLibraryApi = {
+  render: Render;
+  cleanup: Cleanup;
+};
+
+let RNTL: TestingLibraryApi | undefined;
+try {
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  RNTL = require('@testing-library/react-native');
+} catch (error) {
+  // Do nothing
+}
+
+let RTL: TestingLibraryApi | undefined;
+try {
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  RTL = require('@testing-library/react');
+} catch (error) {
+  // Do nothing
+}
+
+export function getRenderFunction(): Render {
+  if (config.render) {
+    if (config.verbose) console.log(`Reassure: using custom 'render' function.`);
+    return config.render;
+  }
+
+  if (config.testingLibrary === 'react-native') {
+    if (!RNTL) {
+      console.error(`Reassure: unable to import '@testing-library/react-native' dependency`);
+      throw new Error(`Unable to import '@testing-library/react-native' dependency`);
```

Comment:
It's redundant to write the same message twice. Can we only throw? It should be visible to the user. If we want to alter the message, let's create a custom Error and adjust stuff there

Reply:
Removed `console.error` calls next to `throws`